frontend/src/PagesAdmin/RecruitmentGangOverviewPage/RecruitmentGangOverviewPage.tsx

```diff
@@ -40,7 +40,7 @@ export function RecruitmentGangOverviewPage() {
   const backendUrl = ROUTES.backend.admin__samfundet_informationpage_changelist;
   const header = (
     <>
-      <Button theme="success" rounded={true} link={ROUTES.frontend.admin_information_create}>
+      <Button theme="success" rounded={true} link={ROUTES.frontend.admin_recruitment_overview}>
```

Comment:
Må fylle recruitmentId i URLen.

```suggestion
      <Button
        theme="success"
        rounded={true}
        link={reverse({
          pattern: ROUTES.frontend.admin_recruitment_overview,
          urlParams: { recruitmentId },
        })}
      >
```

Reply:
Did it